versions/development/SPEC.md

```diff
@@ -1082,7 +991,7 @@ Therefore to ensure that a WDL is portable when using `glob()`, a docker image s
 
 ### String Interpolation
 
-Within tasks, any string literal can use string interpolation to access the value of any of the task's inputs.  The most obvious example of this is being able to define an output file which is named as function of its input.  For example:
+Any string literal can use string interpolation to access the value of any expression. The most obvious example of this is being able to define an output file which is named as function of its input. This uses the same syntax and semantics as expression placeholders in the command section of a task. For example:
```

Comment:
@mlin @cjllanwarne can you please confirm that cromwell or miniwdl implements this feature

Reply:
Yes-ish...miniwdl honors interpolations in any string literal except those in meta/parameter_meta sections

Reply:
Yes, should be the same in Cromwell